docs/agent/hostname-resolution.md

```diff
@@ -33,3 +45,9 @@ If you're affected by this change, we recommend that you take the following acti
 
 - Upgrading from Agent v6 < 6.6 to Agent >= v6.6: If you wish to keep the behavior of Agent v6 (<6.6) for now, set hostname_fqdn to false. We recommend you switch hostname_fqdn to true whenever possible.
 
+### Windows
+
+By default, the recommended action is to do nothing. This will preserve the existing behavior, especially if upgrading from Agent v5.  
```

Comment:
it's a bit weird that our recommended action is to keep the deprecation message 🤔

I think we should encourage users to set the flag to `true` if they're affected (if/when they're ready to see the host names change, we should give some details on what that entails), what do you think?